src/components/PullRequestsTable/PullRequestsTable.tsx

```diff
@@ -14,14 +14,60 @@
  * limitations under the License.
  */
 import React, { FC, useState } from 'react';
-import { Typography, Box, ButtonGroup, Button } from '@material-ui/core';
+import { Typography, Box, ButtonGroup, Button, makeStyles } from '@material-ui/core';
 import GitHubIcon from '@material-ui/icons/GitHub';
 import { Table, TableColumn } from '@backstage/core';
 import { useProjectName } from '../useProjectName';
 import { usePullRequests, PullRequest } from '../usePullRequests';
 import { PullRequestState } from '../../types';
 import { Entity } from '@backstage/catalog-model';
 
+const useStyles = makeStyles(() => ({
+  status: {
+    fontWeight: 500,
+    '&::before': {
+      width: '0.7em',
+      height: '0.7em',
+      display: 'inline-block',
+      marginRight: 8,
+      borderRadius: '50%',
+      content: '""',
+    },
+  },
+  open: {
+    '&::before': {
+      backgroundColor: '##28A745',
+    },
+  },
+  closed: {
+    '&::before': {
+      backgroundColor: '#6F42C1',
+    },
+  },
+}));
+
+const StatusOpen: FC<{}> = props => {
+  const classes = useStyles(props);
+  return (
+    <span
+    className={`${classes.status} ${classes.open}`}
+      aria-label="Status Closed"
+      {...props}
+    />
+  );
+};
+
+const StatusClosed: FC<{}> = props => {
```

Comment:
Does the API tell you if the PR is merged? If so, there should be three states: open (green), closed and merged (purple), closed but not merged (red).

If you can find the GitHub icons somewhere, and you can add them without adding too much size to the bundle, please use them instead of circles.

Reply:
@dtuite ok, Pull request updated. Now it looks like this:

![chrome_2tEGzubMjo](https://user-images.githubusercontent.com/36006588/98485327-009f5900-2216-11eb-8bdb-3e696151fc84.png)

![chrome_RJ4tLjDacJ](https://user-images.githubusercontent.com/36006588/98485352-1f055480-2216-11eb-8095-d8ce82fb705d.png)

Reply:
Pull requests updated.
Now it looks like this:

![chrome_J8CiSP27rS](https://user-images.githubusercontent.com/36006588/98485590-d64e9b00-2217-11eb-9015-41d23cd7c6a2.png)

![chrome_I1ozQHlVwM](https://user-images.githubusercontent.com/36006588/98485599-e49cb700-2217-11eb-9250-de632ca20c53.png)